<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ipv6QtClass</class>
 <widget class="QDialog" name="ipv6QtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ipv6Qt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ipv6qt.qrc">
    <normaloff>:/icons/icons/icon.png</normaloff>:/icons/icons/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
      <item>
       <widget class="QPushButton" name="bt_init">
        <property name="text">
         <string>初始化</string>
        </property>
        <property name="icon">
         <iconset resource="ipv6qt.qrc">
          <normaloff>:/icons/icons/icon.png</normaloff>:/icons/icons/icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_chooseNet">
        <property name="text">
         <string>确定</string>
        </property>
        <property name="icon">
         <iconset resource="ipv6qt.qrc">
          <normaloff>:/icons/icons/xx.jpg</normaloff>:/icons/icons/xx.jpg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>prot:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_port"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bt_getip">
           <property name="text">
            <string>获取服务器地址</string>
           </property>
           <property name="icon">
            <iconset resource="ipv6qt.qrc">
             <normaloff>:/icons/icons/icon.png</normaloff>:/icons/icons/icon.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bt_close">
           <property name="text">
            <string>C L O S E</string>
           </property>
           <property name="icon">
            <iconset resource="ipv6qt.qrc">
             <normaloff>:/icons/icons/xx.jpg</normaloff>:/icons/icons/xx.jpg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ipv6qt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bt_close</sender>
   <signal>clicked()</signal>
   <receiver>ipv6QtClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
